tools/c7n_mailer/c7n_mailer/record.py

```diff
@@ -13,6 +13,14 @@
 # limitations under the License.
 import json
 from cStringIO import StringIO
+from dateutil import parser
+from pytz import timezone
+
+
+def date_time_format(utc_str, tz_str='US/Eastern', format='%Y %b %d %H:%M %Z'):
```

Comment:
this didn't need a pytz dep, dateutil has most of the common ones. the pytz db is more extensive wrt to historical anomalies, but its unclear thats its actually needed for notifications vs the builtins in dateutil.